src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -38,4 +41,11 @@ public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto subur
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    // TODO: <future function> use Spring Security to check if user is logged in
+    @PostMapping("/my/orders/open")
```

Comment:
if this api is to get open orders, it should be like:
- GET method
- `{userId}/orders?status=open`